app/services/form-code-manager.js

```diff
@@ -4,48 +4,59 @@ export default class FormCodeManagerService extends Service {
   startVersion = -1;
 
   formCodeHistory = [];
-  version = this.startVersion; // -1 so start version is 0
+  latestVersion = this.startVersion; // -1 so start version is 0
   referenceVersion = this.startVersion;
 
-  getLatestVersion() {
-    return this.version;
+  getTtlOfLatestVersion() {
+    return this.#getTtlOfVersion(this.latestVersion);
   }
 
-  pinLatestVersionAsReferenceTtl() {
-    this.referenceVersion = this.getLatestVersion();
+  getTtlOfReferenceVersion() {
+    return this.#getTtlOfVersion(this.referenceVersion);
   }
 
-  getHistory() {
-    return this.formCodeHistory;
-  }
-
-  clearHistory() {
-    this.formCodeHistory = [];
-    this.version = this.startVersion;
+  pinLatestVersionAsReference() {
+    this.referenceVersion = this.latestVersion;
   }
 
   addFormCode(ttl) {
     if (this.isTtlTheSameAsLatest(ttl)) {
       return;
     }
 
-    this.version++;
-    this.formCodeHistory[this.version] = ttl;
+    this.latestVersion++;
+    this.formCodeHistory[this.latestVersion] = ttl;
   }
 
   isTtlTheSameAsLatest(compareTtl) {
-    const latestTtl = this.formCodeHistory[this.getLatestVersion()];
+    const latestTtl = this.formCodeHistory[this.latestVersion];
 
     return compareTtl == latestTtl;
   }
 
   isLatestDeviatingFromReference() {
-    if (this.getLatestVersion() == this.referenceVersion) {
+    if (this.latestVersion == this.referenceVersion) {
       return false;
     }
 
     return !this.isTtlTheSameAsLatest(
       this.formCodeHistory[this.referenceVersion]
     );
   }
+
+  clearHistory() {
+    this.formCodeHistory = [];
+    this.latestVersion = this.startVersion;
+  }
+
+  #getTtlOfVersion(version) {
```

Comment:
Typo?
```suggestion
  getTtlOfVersion(version) {
```

Reply:
The method is not used outside of the class so i made it private :) No usecases for it to call it directly atm